docs/yaml/elements/street_segment.yaml

```diff
@@ -112,6 +112,22 @@ tags:
     **StartHouseNumber**, the implementation should ignore the element containing
     it.
   type: xs:integer
+- _name: HouseNumberPrefix
```

Comment:
See other comments about needing to resolve issues around #162 before adding this to the spec.